content/part1.md

```diff
@@ -154,11 +157,11 @@ func main() {
 }
 ```
 
-## Adding and Removing Elements from the Cart
+## Adding and removing elements from the cart
 
-In order to support adding and removing elements from the cart, the workflow needs to respond to different types of signals.
-Signals let you send arbitrary signals to workflows.
-The below code listens to a signal channel for messages that either add or remove items from a shopping cart.
+In order to support adding and removing elements from the cart, the Workflow needs to respond to different types of signals.
+Signals let you send arbitrary signals to Workflows.
```

Comment:
> Signals let you send arbitrary signals to Workflows.
This doesn't sound right - did you mean something else? A signal channel?

Reply:
ha, very good catch. cc @vkarpov15

Reply:
Yeah this sentence is awkward. What do you think of "Signals are a way to notify Workflows of external events"?

Reply:
Yeah that sounds better, @vkarpov15.